format/apple/bplist/bplist.md

```diff
@@ -0,0 +1,60 @@
+### Show full decoding
+```sh
+$ fq d Info.plist
+```
+
+### Timestamps
+Timestamps in Apple Binary Property Lists are encoded as Cocoa Core Data
+timestamps, where the raw value is the floating point number of seconds since
+January 1, 2001. By default, `fq` will render the raw floating point value. In
+order to get the raw value or string description, use the `todescription`
+function, you can use the `tovalue` and `todescription` functions:
+
+```sh
+$ fq 'torepr.SomeTimeStamp | tovalue' Info.plist
+685135328
+
+$ fq 'torepr.SomeTimeStamp | todescription' Info.plist
+"2022-09-17T19:22:08Z"
+```
+
+
+### Get JSON representation
+```sh
+$ fq torepr com.apple.UIAutomation.plist
+{
+  "UIAutomationEnabled": true
+}
+```
+
+### Decoding NSKeyedArchiver serialized objects
+
+A common way that Swift and Objective-C libraries on macOS serialize objects
+is through the NSKeyedArchiver API, which flattens objects into a list of elements
+and class descriptions that are reconstructed into an object graph using CFUID
+elements in the property list. `fq` includes a function, `from_ns_keyed_archiver`,
+which will rebuild this object graph into a friendly representation. The `bplist`
+data must first be passed through `torepr` to create a JSON representation.
```

Comment:
One could in `from_ns_keyed_archiver` check if the input is a "bplist" decode value and do `torepr` autoamatically (something like `if _exttype == "decode_value" and format == "bplist" then torepr end ...`) but maybe the function is already getting complicated. Maybe possibly for later?

Reply:
I like it, makes things more concise. I'll put that in